reference/7.3/Microsoft.PowerShell.Core/About/about_PowerShell_Editions.md

```diff
@@ -55,10 +55,10 @@ $PSVersionTable
 ```Output
 Name                           Value
 ----                           -----
-PSVersion                      7.2.0
+PSVersion                      7.3.0-preview.2
```

Comment:
Nit: Should this be `preview.3`? My prompt is telling me the newer version is available.

```suggestion
PSVersion                      7.3.0-preview.3
```